best-practices/native/react-native/deployment/ios/README.md

```diff
@@ -1,79 +1,27 @@
 # iOS Deployment
 
+Example React Native Projects which have been deployed:
+  * Row Forge: https://github.com/Shift3/row-forge-ns/wiki
```

Comment:
Actually this Nativescript project.

Reply:
Well darn. Thanks for letting me know, I should have double checked. Im glad you reviewed!

Reply:
fixed via: a7223219c6f97731bf007cdadbc06baca9774fb7